README.md

```diff
@@ -11,7 +11,6 @@ This repository hosts the Kubernetes Python Operator for OIDC Gatekeeper
 The OIDC Gatekeeper Operator may be deployed using the Juju command line as follows
 ```bash
 juju deploy oidc-gatekeeper
-juju config oidc-gatekeeper client-secret=<client-secret> public-url=http://<public-url>
```

Comment:
what about adding --trust?

Reply:
It should be covered here https://github.com/canonical/oidc-gatekeeper-operator/pull/166